File: ex1.py
def diamond(rows):
    for i in range(0, rows):
        for j in range(0, rows-i-1):
            print(end=" ")
        for j in range(0, i+1):
            print("*",end=" ")
        print()
    for i in range(0, rows - 1):
        for j in range(0, i + 1):
            print(end=" ")
        for j in range(0,rows-i-1):
            print("*",end=" ")
        print()

rows = int(input("enter nb of rows"))
diamond(rows)

File: ex1.js
var rows = parseInt(prompt("Enter the number of rows:"));

function diamond(rows){
    for(var i = 0; i < rows; i++){
        for(var j = 0; j < rows - i - 1; j++){
            document.write("&nbsp;&nbsp;");
        }
        for (var j = 0; j < 2 * i + 1; j++) {
            document.write("*");
        }
        document.write("<br>");
    }
    for(var i = rows - 2; i >= 0; i--){
        for(var j = 0; j < rows - i - 1; j++){
            document.write("&nbsp;&nbsp;");
        }
        for (var j = 0; j < 2 * i + 1; j++) {
            document.write("*");
        }
        document.write("<br>");
    }
}
diamond(rows);

